docs/core/preview3/tools/csproj.md

```diff
@@ -22,84 +22,148 @@ see [the MSBuild project file](https://docs.microsoft.com/visualstudio/msbuild/m
 
-Alternatively, the element can contain:
+Alternatively, the attribute can contain:
 
-* None – none of the assets are used.
-* All – all assets are used.
+* `None` – none of the assets are used.
+* `All` – all assets are used.
 
 ### DotnetCliToolReference
-`<DotnetCliToolReference>` element specifies the CLI tool that the user wants restores in the context of the project. It is 
+`<DotnetCliToolReference>` item element specifies the CLI tool that the user wants to restore in the context of the project. It is 
 a replacement for the `tools` node in `project.json`. 
 
+```xml
+<DotnetCliToolReference Include="<package-id>" Version="" />
+```
+
 #### Version
-`<Version>` specifies the version of the package to restore. The element respect the rules of the NuGet versioning scheme.
+`Version` specifies the version of the package to restore. The attribute respect the rules of the NuGet versioning scheme.
 
 ### RuntimeIdentifiers
 The `<RuntimeIdentifiers>` element lets you specify a semicolon-delimited list of [Runtime Identifiers (RIDs)](../../rid-catalog.md) for the project. 
-RIDs enable publishing self-contained deployments. 
\ No newline at end of file
+RIDs enable publishing a self-contained deployments. 
+
+```xml
+<RuntimeIdentifiers>win10-x64;osx.10.11-x64;ubuntu.16.04-x64</RuntimeIdentifiers>
+```
+
+
+### RuntimeIdentifier
+The `<RuntieIdentifier>` element allows you to specify only one [Runtime Identifier (RID)](../../rid-catalog.md) for the project. RIDs enable publishing a self-contained deployment. 
+
+```xml
+<RuntimeIdentifier>ubuntu.16.04-x64</RuntimeIdentifier>
+```
+
+
+### PackageTargetFallback 
+The `<PackageTargetFallback>` element allows you to specify a set of compatible targets to be used when restoring packages. It is designed to allow packages that use the dotnet TxM to operate with packages that don't declare a dotnet TxM. If your project uses the dotnet TxM then all the packages you depend on must also have a dotnet TxM, unless you add the `<PackageTargetFallback>` to your project in order to allow non dotnet platforms to be compatible with dotnet. 
+
+The following example provides the fallbacks for all targets in your project: 
+
+```xml
+<PackageTargetFallback>
+    $(PackageTargetFallback);portable-net45+win8+wpa81+wp8
+</PackageTargetFallback >
+```
+
+The following example specifies the fallbacks only for the `netcoreapp1.0` target:
+
+```xml
+<PackageTargetFallback Condition="'$(TargetFramework)'=='netcoreapp1.0'">
+    $(PackageTargetFallback);portable-net45+win8+wpa81+wp8
+</PackageTargetFallback >
+```
+
+## NuGet metadata properties
+With the move to MSbuild, we have moved the input metadata that is used when packing a NuGet package from project.json to csproj files. The inputs are MSBuild properties. The following is the list of properties that are used as inputs to the packing process when using the `dotnet pack` command or the `Pack` MSBuild target that is part of the SDK. 
+
+### IsPackable
```

Comment:
this is not looking good on the built topic. probably would be better to just have a simple list for now until we describe these, correct?